project/src/views/new-evaluation/dropDownSelector.js

```diff
@@ -0,0 +1,25 @@
+const DropdownSelector = function(options = {}) {
+  
+  const arr = [`<legend class="field-title field-title-skin">${options.legends}</legend>`];
+  const constant = options.items.map((el) => {
```

Comment:
Yes, but what does this constant represent? What is its purpose in life? What did it want to be when it grew up?

Reply:
it will grow up to be the constant holding a string full of html, but I guess its name is not ok.  I'll change it